lib/plug/conn.ex

```diff
@@ -1205,6 +1219,13 @@ defmodule Plug.Conn do
       :nomatch -> value
     end
   end
+
+  defp request_url_port(:http, 80), do: ""
+  defp request_url_port(:https, 443), do: ""
+  defp request_url_port(_, port), do: [?:, to_string(port)]
```

Comment:
Will do, thanks. Because it saves the indirection of figuring out how to stringify it?